composites/AlgoliaSearch/SearchResults.js

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+import PropTypes from "prop-types";
+import a11ySpeak from "a11y-speak";
+import { injectIntl, intlShape, defineMessages } from "react-intl";
+import styled from "styled-components";
+import { ZebrafiedListTable } from "../basic/Table/ListTable";
+import { Row } from "../basic/Table/Row";
+import colors from "../../style-guide/colors.json";
+
+const messages = defineMessages( {
+	noResultsText: {
+		id: "searchresults.noresultstext",
+		defaultMessage: "No results found.",
+	},
+	foundResultsText: {
+		id: "searchresults.foundresultstext",
+		defaultMessage: "Found {resultsCount} { resultsCount, plural, one {result} other {results} }",
+	},
+} );
+
+/**
+ * The title of the search result item.
+ */
+const SearchResultTitle = styled.p`
+	padding-left: 10px;
+	margin: 0;
+	font-size: 1em;
+	font-weight: normal;
+`;
+
+/**
+ * The link to the search result.
+ */
+const SearchResultLink = styled.a`
+	color: ${ colors.$color_black };
+	padding: 10px;
+	
+	&:hover, &:focus {
+		color: ${ colors.$color_pink_dark };
+	}
+`;
+
+/**
+ * Returns a single search result item.
+ *
+ * @param {Object} props Properties of the SearchResult component.
+ * @returns {ReactElement} The rendered search result.
+ */
+export function SearchResult( props ) {
+	let post = props.post;
+
+	return (
+		<Row {...props}>
+			<SearchResultLink href={ post.permalink } onClick={ props.handler }>
+				<SearchResultTitle>{ post.post_title }</SearchResultTitle>
+			</SearchResultLink>
+		</Row>
+	);
+}
+
+SearchResult.propTypes = {
+	handler: PropTypes.func.isRequired,
+	post: PropTypes.object.isRequired,
+};
+
+/**
+ * Wraps the search results.
+ */
+const SearchResultsWrapper = styled.div`
+	margin-top: 20px;
+	width: 100%;
+	display: block;
```

Comment:
div elements are natively display: block and full width, consider to remove these two properties.